### DISCUSSION TITLE
tui helptext: sections

### DISCUSSION THREAD

**Discussion by jtagcat** [dundee/gdu#70] [Ideas]
Right now:
```
  │    up, down, k, j    Move cursor up/down                                     │
  │  pgup, pgdn, g, G    Move cursor top/bottom                                  │
  │   enter, right, l    Select directory/device                                 │
  │           left, h    Go to parent directory                                  │
  │                 d    Delete selected file or directory                       │
  │                 e    Empty selected file or directory                        │
  │                 v    Show content of selected file                           │
  │                 i    Show info about selected item                           │
  │                 r    Rescan current directory                                │
  │                 a    Toggle between showing disk usage and apparent size     │
  │                 c    Show/hide file count                                    │
  │                 n    Sort by name (asc/desc)                                 │
  │                 s    Sort by size (asc/desc)                                 │
  │                 C    Sort by file count (asc/desc)                           │
```
```
Proposed: (notice 'Item under cursor' refers to the move cursor up/down, not selection, what is not explained elsewhere)
```
      up, down, k, j    Move cursor up/down                                     
    pgup, pgdn, g, G    Move cursor top/bottom                                  
     enter, right, l    Select directory/device                                 
             left, h    Go to parent directory                                  

                   r    Rescan current directory                                
                   a    Toggle between showing disk usage and apparent size     
                   c    Show/Hide file count                                    
    Item under cursor:
                   d    Delete                       
                   e    Empty                        
                   v    Show contents                           
                   i    Show [info] metadata                           
    Sort by (twice toggles asc/desc):
                   n    name                                 
                   s    size                                 
                   C    file count                           
```

**Comment by dundee** [comment]
I like the idea! Thanks for suggestion!

**Reply by dundee** [reply]
I will include it in the next minor release.
